packages/forms/src/model.ts

```diff
@@ -1869,7 +1869,7 @@ export class FormGroup extends AbstractControl {
   }
 
   /** @internal */
-  _reduceChildren(initValue: any, fn: Function) {
+  _reduceChildren<T>(initValue: T, fn: (acc: T, control: AbstractControl, name: string) => T) {
```

Comment:
I believe we could also specify the return type?
```suggestion
  _reduceChildren<T>(initValue: T, fn: (acc: T, control: AbstractControl, name: string) => T): T {
```